main.tf

```diff
@@ -192,13 +196,26 @@ module "vpc" {
 ################################################################################
 # ALB
 ################################################################################
+
+data "aws_lb" "alb" {
+  arn = local.alb_arn
+}
+
+data "aws_lb_listener" "https_listener" {
+  load_balancer_arn = local.alb_arn
+  port              = 443
+}
+
 module "alb" {
-  source  = "terraform-aws-modules/alb/aws"
-  version = "v6.5.0"
+  source = "git@github.com:endrec/terraform-aws-alb.git?ref=byo"
+  #  source  = "terraform-aws-modules/alb/aws"
```

Comment:
we should leave this as the ALB module managed under this organization

Reply:
Sure we should, but I don't know the version number till terraform-aws-modules/terraform-aws-alb#227 is released.